tests/unit/test_cli_controller.py

```diff
@@ -144,3 +151,51 @@ def test_delete_many_no_id():
     result = runner.invoke(app, ["delete", "many"])
     assert result.exit_code == 3
     assert "Please provide at least 2 task id's to delete." in result.stdout
+
+
+# Sample data to be returned by the mock read_tasks_file function
+sample_tasks = [
+    {
+        "UUID": "123",
+        "name": "Task One",
+        "date": "2023",
+        "priority": "Low"
+    }
+]
+
+
+@pytest.fixture
+def mock_read_tasks_file():
+    """
+    This will mock the read_tasks_file function.
+    """
+    with patch("todo_cli.cli.cli_controller.read_tasks_file") as mock:
+        mock.return_value = sample_tasks
+        yield mock
```

Comment:
Let me preface this with saying: This looks good and will be perfectly fine.

But I wonder if we are breaking the pattern a bit here, when looking at all the other tests we have taken a completely different approach here and I wonder why. If the case was that it would not work with how we run our other tests, then fine but then we should refactor all of them to match this pattern.

I would like to know if there was specific reason for doing this or if you just find this easier, if you find this easier. I think for consistency we should refactor the others as well.